contrib/cirrus/runner.sh

```diff
@@ -140,11 +140,13 @@ exec_container() {
     done <<<"$(passthrough_envars)"
 
     # VM Images and Container images are built using (nearly) identical operations.
+    # port 56789 is our local image-cache registry
     set -x
     # shellcheck disable=SC2154
     exec bin/podman run --rm --privileged --net=host --cgroupns=host \
         -v `mktemp -d -p /var/tmp`:/var/tmp:Z \
         --tmpfs /tmp:mode=1777 \
+        --expose 56789 \
```

Comment:
this does not do anything, not sure if you were expecting anything with that or if this serves documentation purposes.

Reply:
I need the container to be able to talk to host:56789. `--port` does the opposite, IIRC: host can talk to container. I'll look at logs and see what happens.

Reply:
The container uses --network=host so it shares the network namespace with the host so from a network POV there should be no functional difference

Reply:
Duh. Thanks, I missed that.